### PR TITLE
laplacian mask

### PR DESCRIPTION
**Pull request by NayeraMohamed**

### CODE REVIEW

File: Task 2/LaplacianSharp.m [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function [M] = LaplacianSharp()
+%we can aslo replace the 9 at the center with a 5
+M = [-1 -1 1; -1 9 -1; -1 -1 1];
```
Comment (NayeraMohamed) [inline Task 2/LaplacianSharp.m]:
```suggestion
M = [-1 -1 -1; -1 9 -1; -1 -1 -1];
```

File: Task 2/LaplacianSharp.m [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function [M] = LaplacianSharp()
+%we can aslo replace the 9 at the center with a 5
```
Comment (NayeraMohamed) [inline Task 2/LaplacianSharp.m]:
```suggestion
%we can also replace the 9 at the center with a 5
```

File: Task 2/LaplacianSharp.m [hunk 3]
```diff
@@ -0,0 +1,4 @@
+function [M] = LaplacianSharp()
+%we can also replace the 9 at the center with a 5
```
Comment (NayeraMohamed) [inline Task 2/LaplacianSharp.m]:
```suggestion
%3x3 constant filter with 5 or 9 at the center
```

File: Task 2/LaplacianSharp.m (hunk 1 above)
Comment (mariammoubarak) [inline Task 2/LaplacianSharp.m]: M = [-1 -1 -1; -1 9 -1; -1 -1 -1 ];

**Pull request merged**